python/py-sacremoses/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        alvations sacremoses 99102ce
+version             0.0.35
+revision            0
+name                py-sacremoses
+
+license             LGPL-2.1+
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python port of Moses tokenizer, truecaser and normalizer
+
+long_description    ${description}
+
+homepage            https://github.com/alvations/sacremoses
```

Comment:
this is the default set by the github PG, so please remove this line

Reply:
Thanks done.